File: springdoc-openapi-webmvc-core/src/test/java/test/org/springdoc/api/app16/SpringDocApp16Test.java
package test.org.springdoc.api.app16;

import org.junit.jupiter.api.Test;
import org.springdoc.core.Constants;
import org.springframework.boot.SpringBootConfiguration;
import org.springframework.test.context.TestPropertySource;
import test.org.springdoc.api.AbstractSpringDocTest;

import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;

@TestPropertySource(properties = "springdoc.api-docs.enabled=false")
public class SpringDocApp16Test extends AbstractSpringDocTest {

    @SpringBootConfiguration
    static class SpringDocTestApp { }

    @Test
    public void testApp() throws Exception {
        mockMvc.perform(get(Constants.DEFAULT_API_DOCS_URL))
                .andExpect(status().isNotFound());
    }
}

File: springdoc-openapi-tests/springdoc-openapi-kotlin-webmvc-tests/src/test/kotlin/test/org/springdoc/api/v30/app16/SpringDocApp16Test.kt
package test.org.springdoc.api.v30.app16

import org.springdoc.core.utils.Constants.SPRINGDOC_DEFAULT_FLAT_PARAM_OBJECT
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.test.context.TestPropertySource
import test.org.springdoc.api.v30.AbstractKotlinSpringDocMVCTest

@TestPropertySource(properties = ["$SPRINGDOC_DEFAULT_FLAT_PARAM_OBJECT=true"])
class SpringDocApp16Test : AbstractKotlinSpringDocMVCTest() {
    @SpringBootApplication
    class DemoApplication
}
